components/tools/OmeroPy/src/omero/plugins/chgrp.py

```diff
@@ -59,6 +59,17 @@ def _pre_objects(self, parser):
             "grp", nargs="?", type=ExperimenterGroupArg,
             help="""Group to move objects to""")
 
+    def is_admin(self, client):
+        # check if the user currently logged is an admin
+        svc = client.sf.getAdminService()
```

Comment:
ok so I could also clean some existing code retrieving the context from the admin service